config/admin_ranks.txt

```diff
@@ -43,15 +45,13 @@ Game Master		= +EVERYTHING *EVERYTHING
 Lazy Master 	= +EVERYTHING -AUTOLOGIN *EVERYTHING
 Three Dog		= -@ -ADMIN +DJ
 
-Head Coder      = +FUN +STEALTH +POSSESS +REJUV +BUILD +SERVER +DEBUG +VAREDIT +SOUND +SPAWN +RIGHTS +DJ +POLL
-Coder			= +DEBUG +VAREDIT +SERVER +SPAWN +POLL -AUTOLOGIN
+Scribe 			= +SERVER +DEBUG +VAREDIT +SPAWN -AUTOLOGIN
+Head Scribe 	= +@ +POLL -AUTOLOGIN
```

Comment:
Doesn't adding +@ give head scribes host powers due to being higher up in the text file?

Why do scribes have autologin but head scribes don't? I mean, that doesn't really matter,  it's just kind of weird both ways.

Reply:
No, afaik +@ pulls permissions from the one directly above it.